playground/mocks/data/idp/idx/authenticator-verification-custom-app-push-reject.json

```diff
@@ -178,7 +178,7 @@
       "key":"custom_app",
       "id":"aut198w4v0f8dr8gT0g4",
       "displayName":"Custom Push",
-      "logo":"https://tc1static.oktacdn.com/assets/img/logos/custompushLogo.svg",
```

Comment:
is this `logo` prop no longer used anywhere?

Reply:
No it is not. SIW Custom push work was partly done long back and it was not used anywhere. during recent development we have decided to change the prop to `logoUri`